src/editor/index.js

```diff
@@ -351,17 +351,25 @@ class Aganippe {
 
   imageClick () {
     const { container, eventCenter } = this
+    const selectionText = node => {
+      const textLen = node.textContent.length
+      operateClassName(node, 'remove', CLASS_OR_ID['AG_HIDE'])
+      operateClassName(node, 'add', CLASS_OR_ID['AG_GRAY'])
+      selection.importSelection({
+        start: textLen,
+        end: textLen
+      }, node)
+    }
+
     const handler = event => {
       const target = event.target
       const markedImageText = target.previousElementSibling
+      const mathRender = isInMathRender(target)
+      const mathText = mathRender && mathRender.previousElementSibling
       if (markedImageText && markedImageText.classList.contains(CLASS_OR_ID['AG_IMAGE_MARKED_TEXT'])) {
-        const textLen = markedImageText.textContent.length
-        operateClassName(markedImageText, 'remove', CLASS_OR_ID['AG_HIDE'])
-        operateClassName(markedImageText, 'add', CLASS_OR_ID['AG_GRAY'])
-        selection.importSelection({
-          start: textLen,
-          end: textLen
-        }, markedImageText)
+        selectionText(markedImageText)
+      } else if (mathText) {
+        selectionText(mathText)
```

Comment:
Maybe it would be a good idea to add a fallback that maybe logs an error.

Reply:
This code will cause JS error?

Reply:
IDK. I mean if neither the first nor the second if is true it will just silently do nothing. I would add a last else statement that throws an error.

Reply:
I think the `last else` will always happen in this code, because we don't always click the image or math (inline or display math).

Maybe treat it silently is OK?

Reply:
If you think it is fine. Just wanted to point it out